geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseQueryPlan.scala

```diff
@@ -30,11 +30,9 @@ sealed trait HBaseQueryPlan extends HBaseQueryPlanType {
 object HBaseQueryPlan {
 
   def explain(plan: HBaseQueryPlan, explainer: Explainer, prefix: String): Unit = {
-    import org.locationtech.geomesa.filter.filterToString
     explainer.pushLevel(s"${prefix}Plan: ${plan.getClass.getName}")
     explainer(s"Table: ${Option(plan.table).orNull}")
     explainer(s"Ranges (${plan.ranges.size}): ${plan.ranges.take(5).map(rangeToString).mkString(", ")}")
-    explainer(s"Client-side filter: ${plan.clientSideFilter.map(filterToString).getOrElse("None")}")
```

Comment:
yeah, accumulo is nice b/c it's all there in the iterator configs. Possibly once we move to coprocessors we'll have something similar for hbase. In both cases though, the final entriesToFeatures is opaque atm.